Run .NET framework CI on GitHub Actions

Pontus4 opened the pull request.

walro commented on .github/workflows/ci-windows.yml:
```diff
@@ -0,0 +1,27 @@
+        run: msbuild Twingly.Search.sln /p:Configuration=Release
+
+      - name: Run tests
+        run: dotnet test ./Twingly.Search.Tests/bin/Release/Twingly.Search.Tests.dll
```
I don't think this does what you hope it would do, check the output at https://github.com/twingly/twingly-search-api-dotnet/runs/1510183393?check_suite_focus=true#step:6:31 :)

Pontus4 replied:
It sure didn't! It looks better now :)

walro commented on .github/workflows/ci-windows.yml:
```diff
@@ -0,0 +1,32 @@
+      - name: Install dependencies
+        run: |
+          nuget restore Twingly.Search.sln
+          nuget install NUnit.Console -Version 3.9.0 -OutputDirectory packages
```
Since this is a new file, it makes sense to me to use latest available: https://www.nuget.org/packages/NUnit.Console/3.11.1

walro replied:
(I know it's 3.9.0 in Mono, but then it was just a mindless port from Travis 😄)

Pontus4 replied:
All right, I'll update

Pontus4 commented on .github/workflows/ci-windows.yml:
```diff
@@ -0,0 +1,32 @@
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup NuGet
```
The tests passed even without this set up. Do you think it should be kept for "good practice" or should I remove it? @walro

walro replied:
If the action is not needed, we should remove it :)

walro commented:
You can add a badge to the README and also drop Appveyor here

walro commented:
> and also drop Appveyor here

Not sure if there some sort of cleanup that needs to be done over at AppVeyor but you could investigate that too :)

Pontus4 commented:
> You can add a badge to the README and also drop Appveyor here

Fixed in 992a22f

> Not sure if there some sort of cleanup that needs to be done over at AppVeyor but you could investigate that too :)

It was. The project is now removed from there 👍

walro reviewed (approved):
LGTM!
